data-service/mvn_dependency_tree.txt

```diff
@@ -1,4 +1,5 @@
 uk.gov.gchq.palisade:data-service:jar:0.4.0-SNAPSHOT
++- com.google.code.gson:gson:jar:2.8.5:compile
```

Comment:
what was the reason for changing the gson version back a bug fix? (2.8.6 -> 2.8.5)

Reply:
2.8.6 doesnt seem to fix the issues, yet 2.8.5 does!

Reply:
Hi ac74475, the shaded hadoop client has a dependency on gson version <= 2.8.5 - there is an interface: com.google.gson.GsonBuilder com.google.gson.GsonBuilder.setLenient() that has been deleted from 2.8.6 onwards. The gson construction requires this hence the reason that we have to include the specific version.